src/main/java/githubstudy/Main.kt

```diff
@@ -3,6 +3,8 @@ package githubstudy
 object Main {
     @JvmStatic
     fun main(args: Array<String>) {
-        println("Hello world!!")
+        for (i in 1..10) {
```

Comment:
@purluno ten times are too many, five times would be good.

```suggestion
        for (i in 1..5) {
```